front/pages/w/[wId]/workspace/index.tsx

```diff
@@ -233,7 +236,11 @@ export default function WorkspaceAdmin({
         </Page.Vertical>
         <Page.Vertical align="stretch" gap="md">
           <Page.H variant="h4">Capabilities</Page.H>
-          <ContentCreationSharingToggle owner={owner} />
```

Comment:
Moved from `ContentCreationSharingToggle` the `ContextItem.List` so each child is only one item, not the full list